Roys/app/src/main/java/com/dwg_karrier/roys/MainActivity.java

```diff
@@ -55,11 +58,16 @@ protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     setContentView(R.layout.activity_main);
 
+
+    dataBaseOpenHelper = new DataBaseOpenHelper(this);
+
+
     Button btnFeedlyAccount = (Button) findViewById(R.id.FeedlyAccountBtn);
     btnFeedlyAccount.setOnClickListener(new Button.OnClickListener() {
       public void onClick(View v) {
         final String URL = "https://cloud.feedly.com/v3/streams/contents?streamId=user/" + ID + "/category/global.all";
-        new GetPageList().execute(URL);
+        new GetPageList(dataBaseOpenHelper).execute(URL);
```

Comment:
I thought that when user push this button multiple times, this object will be made multiple times. so I bring the `new DataBaseOpenHelper(this)` code out of onClink function. `dataBaseOpenHelper` variable is used only here at this activity. It this situation is okay, I move the initial part to `onClick`!